### PR TITLE
Update mermaid entrypoint diagrams with current state

### PR DESCRIPTION
**Pull request by merelcht**
## Description
Follow up on https://github.com/kedro-org/kedro/pull/4013
Changed all diagrams in `docs/diagrams` to be up to date with the current flows. 

## Developer Certificate of Origin
We need all contributions to comply with the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). All commits must be signed off by including a `Signed-off-by` line in the commit message. [See our wiki for guidance](https://github.com/kedro-org/kedro/wiki/Guidelines-for-contributing-developers/).

If your PR is blocked due to unsigned commits, then you must follow the instructions under "Rebase the branch" on the GitHub Checks page for your PR. This will retroactively add the sign-off to all unsigned commits and allow the DCO check to pass.

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes
- [ ] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

**Review by astrojuanlu — COMMENTED** [review]
We need to re-render them right? https://github.com/kedro-org/kedro/wiki/Render-Mermaid-diagrams

**Comment by merelcht** [comment]
> We need to re-render them right? https://github.com/kedro-org/kedro/wiki/Render-Mermaid-diagrams

Do we? These are not inside the docs. Last time I also just added them like this. If you open the file you'll see the diagram: https://github.com/kedro-org/kedro/blob/602bc26b98ca03994ce7a85b85ef3a874cc233e5/docs/diagrams/installed-kedro-project.md

File: docs/diagrams/kedro-no-project.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-    participant kedro as "$ kedro"
-    participant entrypoint as "setup.py\nkedro = kedro.framework.cli:main"
-    participant init_plugins as "Kedro Plugins\nentry_point = kedro.init"
-    participant kedro_cli as "Kedro CLI\nglobal commands\ninfo, new, docs, starter"
-    participant global_plugins as "Kedro Plugins\nentry_point = kedro.global_commands"
-    participant pyproject.toml as "Current directory\npyproject.toml"
-    participant click as "Click"
+    participant kedro as $ kedro
+    participant entrypoint as pyproject.toml <br> kedro = "kedro.framework.cli:main"
+    participant init_plugins as Kedro Plugins <br> [project.entry-points."kedro.init"]
```
Comment (ankatiyar) [inline docs/diagrams/kedro-no-project.md:7]: Is this a valid entrypoint? I was trying to find info on it but I couldn't.

**Comment by astrojuanlu** [comment]
Sorry I got confused 👍🏼 Thanks!

File: docs/diagrams/kedro-no-project.md (hunk 1 above)
Comment (merelcht) [reply docs/diagrams/kedro-no-project.md:7]: I took me a while to figure this out as well. It's part of the entrypoint groups:

```
``

Comment (ankatiyar) [reply docs/diagrams/kedro-no-project.md:7]: Oh TIL! Thanks!

**Review by ankatiyar — APPROVED** [review]
LGTM!

File: docs/diagrams/installed-kedro-project.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ```mermaid
 sequenceDiagram
-    title "Installed Kedro project"
+    title Installed Kedro project
 
-    participant script as "third-party Python script"
-    participant curr_dir as "Directory with Kedro conf/ in it"
-    participant session as "KedroSession"
+    participant script as third-party Python script
```
Comment (ElenaKhaustova) [inline docs/diagrams/installed-kedro-project.md:5]:
```suggestion
    participant script as Third-party Python script
```

Seems like the rest blocks start with capital letter

**Review by ElenaKhaustova — APPROVED** [review]
Thank you, @merelcht!

Left one neat comment.

**Pull request merged**